src/context/videoContext.js

```diff
@@ -3,7 +3,7 @@ import { videoReducer } from "../reducer/videoReducer";
 const VideoContext = createContext();
```

Comment:
File naming suggestions,
I think you should not use the naming `videoContext` here, as in javascript usually, we name a function like this. And in this file nothing exist with the name `videoContext`.
I think, you can name the way Tanay had suggested, `video-context`, or anything else you find good other than this.

Reply:
made the change